chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -381,10 +381,12 @@ section-button-remove =
     .tooltiptext = { general-remove }
 section-button-add =
     .tooltiptext = { general-add }
-section-button-expand =
-    .tooltiptext = Expand section
-section-button-collapse =
-    .tooltiptext = Collapse section
+section-button-twisty =
+    .tooltiptext = { $status ->
+        [expand] Expand
+        [collapse] Collapse
+        *[other] Toggle
```

Comment:
I know we've had an issue with unspecified values in Fluent, but do we actually need to have `other`, or is the issue just not having a [default](https://mozilla-l10n.github.io/localizer-documentation/tools/fluent/basic_syntax.html#selectors-and-plurals) via `*`? Because if we're always passing a status, it seems like there shouldn't be a need for `other` — we could just set, say, "Collapse" as the default and hopefully it would never actually show up incorrectly (and if it did it would probably only be for a split second, which is not much less weird than seeing "Toggle" briefly).

Reply:
We don't strictly need `other` - it was more of "catch all" case to have a somewhat reasonable placeholder in case if, say, this `l10n-id` is used in some other component in the future and `status` is not passed. It's very much not necessary now, so I just removed it and made `*[collapse]` the default one.